docs/sites.yml

```diff
@@ -3154,3 +3154,16 @@
   built_by: Gabe Ragland
   built_by_url: https://twitter.com/gabe_ragland
   featured: false
+- title: Disrupting Nate
+  description: >
+    Ketogenic Diet, Podcasts, and Blockchain.
+  url: https://www.disruptingnate.com/
+  main_url: https://disruptingnate.com/
+  categories:
```

Comment:
These categories need to be indented a level (thus the failing CircleCI lint job!)

Reply:
Updated